aerleon/lib/pcap.py

```diff
@@ -266,7 +276,7 @@ def _GeneratePortStatement(self, ports, direction):
                 conditions.append('%s portrange %s-%s' % (direction, port_tuple[0], port_tuple[1]))
         return Term.JoinConditionals(conditions, 'or')
 
-    def _GenerateTcpOptions(self, options):
+    def _GenerateTcpOptions(self, options: List[Union[Any, str]]) -> str:
```

Comment:
This is just `List[str]` since it only is ever called with Term.options.

Reply:
Done